src/synthesizer/load_data/load_simba.py

```
@@ -21,146 +21,195 @@ def load_Simba(
+        glist_end_all = hf[f"{obj_str}/glist_end"][:]
+
+        centres_all = hf[f"{obj_str}/pos"][:]
+
+    # Get the particle indices for the selected galaxies
+    if object_indexes is False:
+        galaxy_indices = np.arange(len(slist_start_all))
+
+        star_particle_indices = slist_all
+        gas_particle_indices = glist_all
+    else:
+     else:
```

Ruff (0.11.9)

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Unexpected indentation

---

Line too long (89 > 79)

(E501)